### PR TITLE
update routes and templates

### PR DESCRIPTION
**Pull request by gmarraro**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "jade": "~1.11.0",
```
Comment (noidontdig) [inline package.json]: We can probably remove jade from the package list

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -1,8 +1,263 @@
...
+  .sponsor {
+    margin-right: 0;
+  }
 
-/*# sourceMappingURL=style.css.map */
\ No newline at end of file
+  .team-image {
+    margin-bottom: -15px;
+  }
+}
```
Comment (noidontdig) [inline public/stylesheets/style.css]: It's best practice to add a newline at the bottom of every file -- you can configure your text editor to do so :)

File: routes/program.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET program page. */
+router.get('/program', function(req, res, next) {
+    res.render('program');
```
Comment (noidontdig) [inline routes/program.js]: Let's keep indentation consistent across the codebase -- 2 spaces is standard but open to discussing!

File: views/footer.pug [hunk 4]
```diff
@@ -0,0 +1,5 @@
+footer
+    .footer-container
```
Comment (noidontdig) [inline views/footer.pug]: Let's keep indentation consistent for our pug templates too -- 2 spaces work for you?

File: views/index.pug [hunk 5]
```diff
@@ -0,0 +1,57 @@
+//- extends layout
+
+html(lang='en')
+  head
+    meta(charset='utf-8')
+    //- meta(name='description' content='The Code Cooperative facilitates prisoner reentry
```
Comment (noidontdig) [inline views/index.pug]: Should this be commented out?

File: views/get-involved.pug [hunk 6]
```diff
@@ -0,0 +1,68 @@
+html(lang='en')
```
Comment (noidontdig) [inline views/get-involved.pug]: Let's create an `application.pug` or `scaffolding.pug` or `main.pug` (or a better name) that includes all the shared scaffolding for the templates. ie everything from `html` to `include header.pug`, and the `footer` block also.

Comment (noidontdig) [reply views/get-involved.pug]: Oh it should go in `layout.pug`!

File: views/index.pug (hunk 5 above)
Comment (gmarraro) [reply views/index.pug]: It throws an error at 'content' when I uncomment it-- not sure why!

**Pull request merged**